pkg/component/etcd/monitoring.go

```diff
@@ -30,6 +30,7 @@ import (
 const (
 	monitoringPrometheusJobEtcdNamePrefix          = "kube-etcd3"
 	monitoringPrometheusJobBackupRestoreNamePrefix = "kube-etcd3-backup-restore"
+	monitoringPrometheusJobDruidNamePrefix         = "etcddruid"
```

Comment:
```suggestion
	monitoringPrometheusJobDruidName         = "etcd-druid"
```

This is no prefix, is it?